gossipd/queries.c

```diff
@@ -666,6 +678,10 @@ const u8 *handle_reply_channel_range(struct peer *peer, const u8 *msg)
 
 	scids = decode_short_ids(tmpctx, encoded);
 	if (!scids) {
+		if (complete == 0)
+			return towire_errorfmt(peer, NULL,
+			                       "No up to date infos about this network: %s",
+			                       tal_hex(tmpctx, msg));
```

Comment:
Should we maybe add the chain hash instead of the entire message they sent us?

Reply:
We send the whole message for all errors, I can make it `chain_hash` instead but that's anyway not very useful to the peer (he kind of errored first)